src/Setup/DevDivVsix/Settings.props

```diff
@@ -0,0 +1,7 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="..\..\..\build\Targets\Settings.props" />
+  <PropertyGroup>
+    <SwixBuildPath>$(NuGetPackageRoot)\microbuild.plugins.swixbuild\$(MicroBuildPluginsSwixBuildVersion)\</SwixBuildPath>
```

Comment:
This version is (at least by name) a different version than what it was previously. Expected?

Reply:
Which name here? Everything should be the same as before.

Reply:
I get it now. Your explanation cleared things up.